services/errors.go

```diff
@@ -28,6 +28,15 @@ var (
 		ErrUnimplemented,
 		ErrUnavailableOffline,
 		ErrKava,
+
+		ErrNoOperations,
+		ErrInvalidCurrencyAmount,
+		ErrUnsupportedCurrency,
+		ErrUnclearIntent,
+		ErrInvalidAddress,
+		ErrInvalidOptions,
+		ErrInvalidMetadata,
+
 		ErrUnsupportedCurveType,
 		ErrPublicKeyNil,
 		ErrInvalidPublicKey,
```

Comment:
`ErrMissingPublicKey` is not included in this list, but is defined on L131

Reply:
👍